### ISSUE TITLE
typingTime issue

### ISSUE THREAD

**Issue by voulgarakis** [marcocesarato/react-native-input-spinner#74]
Hi all,

I try use typingTime prop, in order to add some delay before the onChange event occurs. But changing the typingTime does not change anything. Does anyone has the same problem?

<InputSpinner
                style={{marginTop: 10}}
                max={100}
                min={0}
                step={1}
                value={chLowerBound}
                fontSize={14}
                showBorder={true}
                rounded={false}
                typingTime={4000}
                inputStyle={{
                  fontFamily: 'SFProDisplay-Semibold',
                }}
                color={paperTheme.colors.primary}
                width={140}
                height={45}
                onChange={num => {
                  if (num !== 0 && (num < 45 || num > 85)) {
                    console.log('val error 1:', num);
                    setChLowerBound(num);
                  } else if (num !== 0 && num >= chUpperBound) {
                    console.log('val error 2:', num);
                  } else if (num !== 0) {
                    // send data to API
                  }
                }}
              />

**Comment by savitrasapre** [comment]
Agreed. I'm having trouble adding a decimal after an integer. Even if I enter 10. it changes it to 10

Even if I log the onChange event, the "." does not get caught and is parsed before it is sent to onChange. Did you figure out a way to get around it?

typingTime does nothing even after I increase the time (in ms)
